### ISSUE TITLE
Preserve character sheet state

### ISSUE THREAD

**Issue by dhleong** [larrychristensen/orcpub#15]
While testing to see if the "round trip" for #4 worked, I noticed that changes to a character sheet aren't actually persisted unless you do so through "edit" mode (but #4 does seem to work when doing that). This means hit points, spell usage, etc. get reset when you close the browser. Is that intended? If not, it seems like dispatching `:save-character` from the `update-character-fx` handler might do the trick.

**Comment by larrychristensen** [comment]
Yes, that's a bug I haven't gotten around to fixing yet. I would love to
see this fixed. I'd also love to see better hit point and hit dice
management. Pretty much everything DnDBeyond is doing was borrowed from my
roadmap, so I'd like to to do everything they are doing and more.
